### ISSUE TITLE
What permissions does the PAT need?

### ISSUE THREAD

**Issue by haberdashPI** [joshuakto/fit#33]
Would be good to add this to the readme. I assume it is all `repo` permissions.

**Comment by igorovic** [comment]
I tested the plugin with a minimal set of permission and limiting access to only one repo. And so far it seems to work. 

![image](https://github.com/user-attachments/assets/7efa47b1-263d-4bbf-9c6c-845bc41da35e)

**Comment by haberdashPI** [comment]
Yes, I agree that does seem to be the requirement. Just needs to be added to the README.

**Comment by haberdashPI** [comment]
No, this issue is not stale.
